backend/substrapp/tests/common.py

```diff
@@ -293,6 +293,10 @@ def __init__(self, status, content):
         self.status_code = status
         self.content = content
 
+    def raise_for_status(self):
+        if self.status_code != 200:
+            raise Exception(f'HTTP Exception : {self.status_code}')
+
```

Comment:
I might be missing something, but doesn't that apply only to tests? I'm not seeing anything checking the response  code of actual requests.

Reply:
Old code ! I will remove it